### PR TITLE
Added "Espeak" tldr page

### PR DESCRIPTION
**Pull request by Fallon9111**
An altered version of the osx 'say.md' file for the espeak command

### CODE REVIEW

File: pages/common/espeak.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# espeak
+
+> Uses text to speech to speak through the default sound device
```
Comment (waldyrious) [inline pages/common/espeak.md]: add hyphens: text-to-speech

File: pages/common/espeak.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Speak a phrase aloud
+
+`espeak "I like to ride my bike."`
+
+- Speak a file aloud
+
+`espeak -f {{filename}}`
+
+- Write speech to a WAV audio file, rather than speaking it directly
```
Comment (waldyrious) [inline pages/common/espeak.md]: "Write speech" sounds awkward. How about "Save output"?

**Comment by waldyrious** [comment]
Nice one. Please take a look at the comments I added inline.

**Comment by Fallon9111** [comment]
I've implemented your suggestions, and also extended the format 'text-to-speech' to osx/say.md for continuity.

**Comment by waldyrious** [comment]
Good call! Looks good to merge, but you'll probably need to remove the index file from this PR, and squash the commits. /cc @igorshubovych

**Comment by igorshubovych** [comment]
Agreed. Please squash commits and remove index.json.

Also please run `make setup` to ensure that you use the latest precommit hooks.

**Comment by igorshubovych** [comment]
Thank you for your contribution!

**Pull request merged**